src/vue/components/character/Characteristic.vue

```diff
@@ -18,6 +18,11 @@ withDefaults(
 		 */
 		canEdit?: boolean;
 
+        /**
+		 * Can the characteristic be used to directly to roll an unskilled check?
+		 */
+		canRollUnskilled?: boolean;
+
```

Comment:
Rather than this, it may be best to just go ahead and let all the Characteristic fields allow an unskilled check; I don't think this needs to be restricted to just Adversaries.

Reply:
Nevermind; since this project isn't using Pinia stores to fetch the relevant Actor cleanly, that would ultimately need us to pass the Actor to the Characteristic component. (Leaving the comment in as part of the review for archival reasons :D)

Reply:
Yeah, I had in mind to allow it for Characters as well but decided not to because of redundancy (in theory a character sheet has all the skills listed so there's no need for an unskilled roll). However, it should be easy to make it work for characters as well and maybe there's value in consistency over redundancy. I think I might expand it to characters in a future PR but I want to ask other peeps for feedback first.